draft-ietf-oauth-identity-chaining.md

```diff
@@ -77,7 +79,7 @@ A client in trust domain A that needs to access a resource server in trust domai
 
 ## Overview
 
-The identity and authorization chaining flow outlined below describes how a combination of OAuth 2.0 Token Exchange {{RFC8693}} and JWT Profile for OAuth 2.0 Client Authentication and Authorization Grants {{RFC7523}} are used to address the use cases identified. The appendix include two additional examples that describe how this flow is used. In one example, the resource server acts as the client and in the other, the authorization server acts as the client.
+The identity and authorization chaining flow outlined below describes how a combination of OAuth 2.0 Token Exchange {{RFC8693}} and JWT Profile for OAuth 2.0 Client Authentication and Authorization Grants {{RFC7523}} are used to address the use cases identified. Section [Examples](#examples) include two additional examples that describe how this flow is used. In one example, the resource server acts as the client and in the other, the authorization server acts as the client.
```

Comment:
https://drafts.oauth.net/oauth-identity-chaining/kelley/add-sender-constraints/draft-ietf-oauth-identity-chaining.html#name-overview this currently shows up as 

> Section [Appendix B](https://drafts.oauth.net/oauth-identity-chaining/kelley/add-sender-constraints/draft-ietf-oauth-identity-chaining.html#examples) includes two additional

Reply:
Actually that's the preview for a different branch and I don't see one for this branch and I don't know why 

https://drafts.oauth.net/oauth-identity-chaining/

Reply:
oh, it's because the "build" for this PR failed 

https://github.com/oauth-wg/oauth-identity-chaining/actions/runs/10864604120/job/30150062041?pr=95

```
...
/github/workspace/draft-ietf-oauth-identity-chaining.xml(654): Warning: Artwork too wide, reducing indentation from 0 to 0
/github/workspace/draft-ietf-oauth-identity-chaining.xml(652): Warning: Figure too wide, reducing indentation from 3 to 1
/github/workspace/draft-ietf-oauth-identity-chaining.xml(716): Warning: Artwork too wide, reducing indentation from 0 to 0
/github/workspace/draft-ietf-oauth-identity-chaining.xml(714): Warning: Figure too wide, reducing indentation from 3 to 1
/github/workspace/draft-ietf-oauth-identity-chaining.xml(232): Warning: Too long line found (L361), 3 characters longer than 72 characters: 
&subject_token_type=urn%3Aietf%3Aparams%3Aoauth%3Atoken-type%3Aaccess_token
lib/trace.sh draft-ietf-oauth-identity-chaining.html -s xml2rfc-html xml2rfc -q --rfc-base-url https://www.rfc-editor.org/rfc/ --id-base-url https://datatracker.ietf.org/doc/html/ --cache=/github/workspace/.refcache --html --css=lib/v3.css --metadata-js-url=/dev/null draft-ietf-oauth-identity-chaining.xml -o draft-ietf-oauth-identity-chaining.html
/github/workspace/draft-ietf-oauth-identity-chaining.xml(13): Warning: Expected a title or title abbreviation of not more than 40 character for the page header, found 56 characters
147:requested_cnf_supported_mtls 
188:  * AS-A MUST verify the proof of possession in the token exchange request. 
485:  
draft-ietf-oauth-identity-chaining.md contains trailing whitespace
*** Run 'make fix-lint' to automatically fix some errors
make[1]: *** [lib/main.mk:202: lint-whitespace] Error 1
make[1]: Target 'lint' not remade because of errors.
rm draft-ietf-oauth-identity-chaining.xml
make: *** [lib/main.mk:16: all] Error 2
```

Reply:
that makes it hard to review

Reply:
I'm struggling with the web interface that I'm stuck using. Will continue working to resolve

Reply:
@bc-pi Pieter figured it out. Take a look now

Reply:
That part doesn't look like it renders like you'd want 
https://drafts.oauth.net/oauth-identity-chaining/add-sender-constrained-tokens/draft-ietf-oauth-identity-chaining.html#name-overview

Reply:
Not sure what you mean. Let's talk next week

Reply:
Maybe just:
```suggestion
The identity and authorization chaining flow outlined below describes how a combination of OAuth 2.0 Token Exchange {{RFC8693}} and JWT Profile for OAuth 2.0 Client Authentication and Authorization Grants {{RFC7523}} are used to address the use cases identified. (#examples) include two additional examples that describe how this flow is used. In one example, the resource server acts as the client and in the other, the authorization server acts as the client.
```